app/index.js

```diff
@@ -493,7 +493,7 @@ class CustomModalStack extends React.Component {
 			</View>
 		);
 
-		if (isAndroid) {
+		if (isAndroid && !isTablet) {
```

Comment:
Hey @Badal1234! It's not correct 😅.
If we're showing this component we have certain that is a tablet, then it'll always evaluates to `true` and never enter on this condition, we're needing this `ScrollView` because we have some modals that have inputs and the keyboard is opened, it was added by #1742 and we can't do this condition here because it'll broken this fix.
Maybe you can use `pageSheet` value to handle the `ScrollView` and only add the `ScrollView` when `isAndroid && !pageSheet`.

Reply:
Thanks @djorkaeffalexandre ,for your review and suggestion.